packages/round-manager/src/features/round/ViewApplicationPage.tsx

```diff
@@ -46,6 +53,67 @@ export default function ViewApplicationPage() {
     }),
   })
 
+
+  const [answerBlocks, setAnswerBlocks] = useState<AnswerBlock[]>();
```

Comment:
@chibie as you had suggested I used useEffect cause but I'm unclear on how to ensure this gets called once after the application.answers have loaded. `answerBlocks ` gets overwritten multiple times and that prevents me from showing the needed data 

The decryption piece works without any issues

cc @shavinac / @dschinkel would love if you guys had some ideas here

URL: http://localhost:3000/#/round/0xb91fec0b68f39cbfdd75e4f08042c60724e1bd3b/application/0x3600000000000000000000000000000000000000000000000000000000000000-0xb91fec0b68f39cbfdd75e4f08042c60724e1bd3b

The IPFS ptr: https://gitcoin.mypinata.cloud/ipfs/bafkreif6f7e2jaa7445jflaintyav5m63gutxl6umpgvxsjhdqsqv326za

Reply:
Thanks @shavinac for helping me out here.
I've cherry-picked your commit onto my branch